Formula/llvm.rb

```diff
@@ -36,13 +36,6 @@ class Llvm < Formula
       url "https://raw.githubusercontent.com/Homebrew/formula-patches/6166a68c/llvm/openmp_arm.patch"
       sha256 "70fe3836b423e593688cd1cc7a3d76ee6406e64b9909f1a2f780c6f018f89b1e"
     end
-
-    # Regression in LLDB by D89156 on MacOS, fixed by D95683.
-    # https://reviews.llvm.org/D95683
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/698a42e95e2c982c1d11f1b7d3293ba3aac677ee/llvm/lldb.patch"
-      sha256 "b0dd8153370de8333dc57f950976bc87413c0b053bd30d9b8b10540094923b88"
-    end
   end
 
   livecheck do
```

Comment:
- - https://wiki.python.org/moin/LocalUserGroups?action=raw

Reply:
what do you mean?